### DISCUSSION TITLE
Unverified - no miners changed?

### DISCUSSION THREAD

**Discussion by Osmodia666** [revoxhere/duino-coin#1414] [Q&A]
Hi, 
my Account (Osmodia) is unverified. I was verified since the beginning of the verification process, and now I am not verified. 
I did not change anything and have alwas used the up to date code.

Thanks.

**Comment by Osmodia666** [comment]
Sadly no one wants to help.

**Comment by Osmodia666** [comment]
Thx for no Help.

**Comment by SoraKohaku** [comment]
> Hi, 
> my Account (Osmodia) is unverified. I was verified since the beginning of the verification process, and now I am not verified. 
> I did not change anything and have alwas used the up to date code.
> 
> Thanks.

Turn on the miners and photo correct way. But I'm still confused if have new miner. Cause my Arduino set already default by code
